apps/ledger-live-desktop/tests/fixtures/common.ts

```diff
@@ -241,10 +242,15 @@ export const test = base.extend<TestFixtures>({
   ],
 });
 
-export async function addTmsLink(ids: string[]) {
+export async function addTestAnnotations(testInfo: TestInfo, ids: string[], name: string) {
   for (const id of ids) {
     await allure.tms(id, `https://ledgerhq.atlassian.net/browse/${id}`);
+    testInfo.annotations.push({ type: "test_key", description: `${id}` });
   }
+  testInfo.annotations.push({
+    type: "test_summary",
+    description: `${name}`,
```

Comment:
I'm not sure whether we need description and test_summary here to be honest

Reply:
It's for Jira, I think it's a good idea to add infos on the test

Reply:
But test case ticket already contains the same info you provide in the description, to me it looks like a duplicate